src/libcore/lib.rs

```diff
@@ -65,6 +65,8 @@
 #![allow(unknown_features)] #![feature(int_uint)]
 #![feature(on_unimplemented)]
 #![deny(missing_docs)]
+// SNAP 9e4e524 remove `allow(unused_mut)` after a snapshot
+#![allow(unused_mut)]
```

Comment:
Now that we have `cfg_attr`, could this be `#![cfg_attr(stage0, allow(unused_mut))]`? Anything with the string "stage0" in it is a pretty easy flag for something to clean out when dealing with snapshots.